pkg/encryption/app.go

```diff
@@ -0,0 +1,136 @@
+package encryption
+
+import (
+	"context"
+	"fmt"
+
+	chartv1 "github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"github.com/giantswarm/k8smetadata/pkg/annotation"
+	"github.com/giantswarm/k8smetadata/pkg/label"
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/encryption-provider-operator/pkg/project"
+)
+
+const (
+	configMapName  = "encryption-config-hasher-chart-values"
+	chartNamespace = "giantswarm"
+
+	appName      = "encryption-config-hasher"
+	appNamespace = "kube-system"
+
+	encryptionConfigHasherVersion = "0.1.1"
+)
+
+var (
+	chartURL = fmt.Sprintf("https://giantswarm.github.io/giantswarm-playground-catalog/encryption-config-hasher-%s.tgz", encryptionConfigHasherVersion)
```

Comment:
Sorry to nitpick, but the should we maybe make a flag out of the catalog? It might very well change in the future.
We can just make a new release, I know, but still.
What about china? Will this work?

Reply:
China also uses GitHub catalogs so it should work, the catalog name can be configurable I will add that.